components/p3a/brave_histogram_rewrite.cc

```diff
@@ -27,18 +27,12 @@ constexpr const char* kBravezationHistograms[] = {
 void DoHistogramBravezation(base::StringPiece histogram_name,
                             base::HistogramBase::Sample sample) {
   if ("Bookmarks.Count.OnProfileLoad" == histogram_name) {
-    int answer = 0;
-    if (0 <= sample && sample < 5)
-      answer = 0;
-    if (5 <= sample && sample < 20)
-      answer = 1;
-    if (20 <= sample && sample < 100)
-      answer = 2;
-    if (sample >= 100)
-      answer = 3;
-
-    UMA_HISTOGRAM_EXACT_LINEAR("Brave.Core.BookmarksCountOnProfileLoad", answer,
-                               3);
+    constexpr int kIntervals[] = {5, 20, 100, 500, 1000, 5000, 10000};
```

Comment:
Question: If sample is 5, current answer is `0`. but previous was `1`.
Which one is correct?

Reply:
the current one, previous was wrong :(
```
Q5: How many bookmarks do you have?
(Brave.Core.BookmarksCountOnProfileLoad)
0-5
6-20
21-100
Over 100
```